crd/awscluster.yaml

```diff
@@ -0,0 +1,80 @@
+# From https://github.com/giantswarm/apiextensions/blob/e11354aee7703fa74c504978a3d4b525c7814730/pkg/apis/infrastructure/v1alpha2/aws_cluster_types.go#L14-L56
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: awsclusters.infrastructure.giantswarm.io
+spec:
+  group: infrastructure.giantswarm.io
+  names:
+    kind: AWSCluster
+    plural: awsclusters
+    singular: awscluster
+  version: v1alpha2
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            cluster:
+              properties:
+                description:
+                  maxLength: 100
+                  type: string
+                dns:
+                  properties:
+                    domain:
+                      type: string
+                    provider:
+                      properties:
+                        master:
+                          properties:
+                            availabilityZone:
+                              type: string
+                            instanceType:
+                              type: string
+                          type: object
+                        region:
+                          type: string
+                      type: object
+                  type: object
+              type: object
+          type: object
+        status:
```

Comment:
Is there a reason why spec and status are in different orders? In the App CR it is the other way around. I would appreciate a bit consistency here. Preferably the status last.

Reply:
Oh, that whole folder is a leftover and must be removed. The ordering has been done by some `yaml.Marshal()`.

Reply:
Removed